core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/ArrayRun.java

```diff
@@ -154,6 +166,31 @@ public void setCompletionDate(LocalDate completionDate) {
     this.completionDate = completionDate;
   }
 
+  @Override
+  public List<FileAttachment> getAttachments() {
+    return attachments;
+  }
+
+  @Override
+  public void setAttachments(List<FileAttachment> attachments) {
+    this.attachments = attachments;
+  }
+
+  @Override
+  public String getAttachmentsTarget() {
+    return "arrayRun";
```

Comment:
Should probably be all lowercase for consistency with the existing `servicerecord`